docs/content/en/docs/how-tos/debug/_index.md

```diff
@@ -43,13 +44,20 @@ representations are untouched.
 
   - Only the `kubectl` and `kustomize` deployers are supported at the moment: support for
     the Helm deployer is not yet available.
-  - Only JVM and NodeJS applications are supported:
+  - File sync is disabled for all artifacts, regardless of whether
```

Comment:
I find this sentence a little hard to read, but can't think of a good alternative.
`File sync is disabled for all artifacts, whether or not they can be configured for debugging?`

Reply:
I'll drop the latter part.